gateway-server/src/main/java/org/apache/knox/gateway/services/token/impl/AliasBasedTokenStateService.java

```diff
@@ -125,6 +120,19 @@ protected boolean isUnknown(final String token) {
     return isUnknown;
   }
 
+  @Override
+  protected void removeToken(final String token) {
+    validateToken(token);
+
+    try {
+      aliasService.removeAliasForCluster(AliasService.NO_CLUSTER_NAME, token);
+      aliasService.removeAliasForCluster(AliasService.NO_CLUSTER_NAME,token + "--max");
+    } catch (AliasServiceException e) {
+      log.failedToUpdateTokenExpiration(e);
```

Comment:
This seems a misleading log message

Reply:
Right, I'll get it fixed, thanks for letting me know @smolnar82